src/haz3lcore/prog/ModelResults.re

```diff
@@ -1,5 +1,13 @@
 open Sexplib.Std;
 
+/* This module is a map used to store ModelResult.t objects
+      where the key is used to distiguish between editors on
```

Comment:
the key distinguishes between different requested evaluations, which may not correspond directly to editors due to splicing